pyramid_oereb/contrib/data_sources/standard/sources/plr.py

```diff
@@ -162,6 +162,7 @@ def from_db_to_view_service_record(self, view_service_from_db, legend_entry_reco
             Config.get('default_language'),
             Config.get('srid'),
             Config.get('proxies'),
+            Config.get('verify_certificate'),
```

Comment:
Where is 'verify_certificate' defined in file 'pyramid_oereb.yml.mako'?

Reply:
at the wms level, like layer opacity - so your following comments are accurate, I have changed the code accordingly.